flask_iiif/restful.py

```diff
@@ -68,22 +56,31 @@ class IIIFImageAPI(Resource):
 
     method_decorators = [
         error_handler,
+        api_decorator,
     ]
 
     def get(self, version, uuid, region, size, rotation, quality,
             image_format):
         """Run IIIF Image API workflow."""
-        # Validate IIIF parameters
-        IIIFImageAPIWrapper.validate_api(
+        api_parameters = dict(
             version=version,
+            uuid=uuid,
             region=region,
             size=size,
             rotate=rotation,
             quality=quality,
             image_format=image_format
         )
+        # Trigger event before proccess the api request
+        iiif_before_process_request.send(api_parameters)
```

Comment:
the first argument (`sender`) should be for sure `self` in order to have access to correct object in signal handler.